pkg/kudoctl/cmd/convert/convert.go

```diff
@@ -48,20 +49,7 @@ func Run(cmd *cobra.Command, args []string, options *Options) error {
 	if e != nil {
 		return e
 	}
-	type P struct {
```

Comment:
this moved to the bundle parsing, this is specific to our bundle format to I think it should be encapsulated there